charts/neuvector-monitor/102.0.8+up2.7.3/Chart.yaml

```diff
@@ -0,0 +1,27 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: NeuVector Monitor
+  catalog.cattle.io/kube-version: '>=1.18.0-0 < 1.29.0-0'
```

Comment:
Should this be?
```
catalog.cattle.io/kube-version: '>=1.18.0-0 < 1.28.0-0'
```

Reply:
@nicholasSUSE NeuVector supports k8s 1.28 so updated the annotation. I tested the chart on downstream cluster with k8s 1.28.6

Reply:
@nicholasSUSE This chart might be used outside Rancher Manager. We offer all the charts via [charts.rancher.io](charts.rancher.io) so users can use them via `helm CLI`. It's up to the feature chart teams to say in which k8s versions the charts might be installed